src/components/coin.ts

```diff
@@ -67,6 +67,7 @@ export const coinBonusMap = new Map<BonusType, Bonus>([
 export interface UserCoinEntry {
   user_id: string;
   balance: number;
+  rank: number;
```

Comment:
Since rank is now manually computed, this doesn't seem to be needed?

Reply:
Fixed it in this commit.